README.md

```diff
@@ -10,24 +10,49 @@
         <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow"><img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=vulnerabilities"/></a>
         <a style="padding: 0 5px" href="https://sonarcloud.io/summary/new_code?id=luno_workflow"><img src="https://sonarcloud.io/api/project_badges/measure?project=luno_workflow&metric=duplicated_lines_density"/></a>
         <a style="padding: 0 5px" href="https://pkg.go.dev/github.com/luno/workflow"><img src="https://pkg.go.dev/badge/github.com/luno/workflow.svg" alt="Go Reference"></a>
+        <a style="padding: 0 5px" href="https://github.com/avelino/awesome-go"><img src="https://awesome.re/mentioned-badge-flat.svg" alt="Mentioned in Awesome Go"></a>
     </div>
 </div>
 
 # Workflow
 
-Workflow is an event driven workflow that allows for robust, durable, and scalable sequential business logic to
-be executed in a deterministic manner.
+**Workflow** is a distributed event driven workflow framework that runs robust, durable, and
+scalable sequential business logic on your services.
+
+**Workflow** uses a [RoleScheduler](https://github.com/luno/workflow/blob/main/rolescheduler.go) to distribute the work
+ across your instances through a role assignment process (similar to a leadership election process but with more than
+ the single role of leader).
+
+**Workflow** expects to be run on multiple instances but can also be run on single
+ instances. Using the above-mentioned [RoleScheduler](https://github.com/luno/workflow/blob/main/rolescheduler.go),
+**Workflow** is able to make sure each process only runs once at any given time
+ regardless if you are running 40 instances of your service or 1 instance.
+
+---
+
+1. [Features](#features)
+2. [Installation](#installation)
+3. [Adapters](#adapters)
+4. [Connectors](#connectors)
+5. [Basic Usage](#basic-usage)
+6. [What is a workflow Run](#what-is-a-workflow-run)
+7. [Hooks](#hooks)
+8. [Configuration Options](#configuration-options)
+9. [Glossary](#glossary)
+10. [Best Practices](#best-practices)
```

Comment:
My experiences with Table of Contents are that they can very easily become outdated: If sections are:
- removed
- added
- renamed
- moved

[GitLab](https://gitlab.com/gitlab-org/gitlab/-/issues/21901) has a nice ` [[_TOC_]]` you can add which auto-generates it, but this isn't supported in GitHub. 

I see from https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/ that you can easily see the Table of Contents from the viewer. Wondering if that is sufficient 🤔

Reply:
Cool will remove. Was a lot of effort just to create and agreed not worth having to try maintain.